package.json

```diff
@@ -220,19 +225,24 @@
       ],
       "view/title": [
         {
-          "command": "pacCLI.authPanel.refresh",
+          "command": "pacCLI.authPanel.newAdminAuthProfile",
           "when": "view == pacCLI.authPanel",
-          "group": "navigation"
+          "group": "navigation@0"
         },
         {
           "command": "pacCLI.authPanel.newDataverseAuthProfile",
           "when": "view == pacCLI.authPanel",
-          "group": "navigation"
+          "group": "navigation@1"
         },
         {
-          "command": "pacCLI.authPanel.newAdminAuthProfile",
+          "command": "pacCLI.authPanel.refresh",
           "when": "view == pacCLI.authPanel",
-          "group": "navigation"
+          "group": "navigation@2"
+        },
+        {
+          "command": "pacCLI.authPanel.clearAuthProfile",
+          "when": "view == pacCLI.authPanel",
+          "group": "navigation@3"
```

Comment:
Hmm.   Perhaps we would to put Clear in the `...` overflow menu by removing the navigation group here?
I don't have particularly strong opinions on either option, but that seems like a better place to me.
@davidjenni , opinions on the placement?